src/pages/lessons/[slug].tsx

```diff
@@ -138,7 +144,7 @@ const Lesson: FunctionComponent<LessonProps> = ({initialLesson}) => {
     free_forever,
   } = lesson
 
-  React.useEffect(() => {
```

Comment:
@DrShpongle please use the `React.useState` syntax. This is to distinguish clearly from React internat hooks and user hooks.